JS/anims-missiles.js

```diff
@@ -1,11 +1,25 @@
-function dropMissile(columnNumber, animationTime) {
+setTimeout(repeatAnims, 500)
+repeatAnims()
+
+function repeatAnims() {
+    setInterval( function () {
+        dropMissile(1000)
+    }, 1000 )
+}
+
+function dropMissile(animationTime) {
+    var columnNumber = randomNumGen()
     $('#pos-'+columnNumber).prepend('<img src="img/missile-drop-down-white.png" class="missile" id="bomb-'+ columnNumber +'">')
 
     $('#bomb-' + columnNumber).animate({
-        top: "+=350"
+        top: "+=550"
     }, animationTime, "linear", function () {
         $(this).remove()
-
     })
 }
-// run on start button
\ No newline at end of file
+
+function randomNumGen() {
+    var randomNum = Math.floor(Math.random() * 4) + 1
```

Comment:
Could use the below:
`Math.ceil(Math.random() * 4)`

Reply:
This has been fixed in 6181f9b